README.md

```diff
@@ -3,53 +3,73 @@ dojos
 
 This is where the Novoda team do all their hacking
 
-# Practicing a new dojo
+# Running a dojo
 
-- find the dojo's module you want to attempt this will be a module under the root dir
-- inside this copy the `default` directory and name it with the date and your name or pairing name
-- this dir you just copied should have instructions to run the dojo as well as any example code
+Find the module of the dojo you want to attempt. This will be a module under the root dir. Inside you will either find the  `default`  directory or a `java` and `swift` directory.
+
+If you only find the  `default` directory, follow [these instructions](https://github.com/novoda/dojos#Updating-a-dojo). Once the folder structure has been updated properly you can continue, make a copy of the default project of whichever language you want to run the kata with.
```

Comment:
> `[these instructions](https://github.com/novoda/dojos#Updating-a-dojo)`

Tip: if it's on the same page you only need to put the anchor: `[these instructions](#Updating-a-dojo)`